spec/support/content_data_api.rb

```diff
@@ -119,6 +119,12 @@ def content_data_api_has_document_types
         stub_request(:get, url).to_return(status: 200, body: body)
       end
 
+      def content_data_api_has_csv
```

Comment:
We should be explicit when setting the fixture for a test. Could you please extract the body?

Reply:
👍